_drafts/2020-01-01-nyu-cs-transfer-perspective.md

```diff
@@ -4,45 +4,37 @@ date: 2020-01-01 6:30:00 -0700
-it was difficult for me to break into the field with such gatekeeping.
+The student body at UMD certainly is stronger in term of programming ability. There is no doubt about that. However, as someone who was new
+to programming and started in college, I noticed certain levels of cockiness, usually from those who learned programming in high school. I was intimidated to ask questions that might sound stupid to these students though those questions might be necessary to my understanding of the topic. Maybe I'm to blame for not being brave enough to ask them. But as someone who was new, it was difficult for me to break into the field with the gatekeepers.
 
-I hope this doesn't come across as slandering UMD. I truly enjoy my time at the school. I learned so much about computer science and build a solid foundation before transferring. I also found great friends and meet brilliant TAs as well as inspiring professors. However, I wasn't
-happy at the school and decided to leave. So, I ensure I made the most out of my education at NYU CS given my knowledge of the courses at UMD. So here's a few things I enjoy about NYU CS and how one can make the most out of the department.
+I hope this doesn't come across as slandering UMD. I truly enjoyed my time at the school. I learned so much about computer science and built a solid foundation before transferring. I also found great friends and met brilliant TAs as well as inspiring professors. However, I wasn't
+happy and decided to leave. By the time I left, I promised myself that I would make the most out of my CS education at NYU. So here're a few things I enjoy about NYU CS and how one can make the most out of your time here.
 
 # Classes
-The CS requirement courses at NYU are lackluster. It doesn't cover enough. Fortunately, the classes needed to cover the foundation are still available. A lot of students point out that the materials covered are not sufficient compared to other institution. However, if you choose the right professor, I believe the materials are quite on par and you will get the most out of it. Yes, yes, CS is difficult and sometimes you want to coast through certain classes. However, if you are serious about your CS education, only way is to put in the work and you will feel rewarding. As a CS student, here's my recommendations to cover the fundamentals:
+The CS requirement courses at NYU are lackluster. It doesn't cover enough. Fortunately, the classes needed to cover the CS foundation are still available. A lot of students point out that the materials covered are not sufficient compared to other institution. However, if you choose the right professor, I believe the materials are quite on par and you will get the most out of it. Yes, yes, CS is difficult and sometimes you want to coast through certain classes. But if you are serious about your CS education, the only way is to put in the work and you will feel rewarding. As a CS student, here're my recommendations:
```

Comment:
Common grammar mistake. When comparing two things, make sure they're the same type. Also the insufficiency is apparent when the comparison happens. I.e. I compared NYU's curriculum to UMD's and when I did, I found NYU's curriculum insufficient.

> that the materials covered are not sufficient compared to other institution

In this case we want to specify that the materials covered are not sufficient *when* compared to the materials of the institution:

> that the material covered is not sufficient when compared to the material covered at other institutions

or more concisely:

> that the material covered is not sufficient when compared to that which is covered at other institutions